ebpf/profiling/Profile.proto

```diff
@@ -84,4 +85,13 @@ message EBPFOnCPUProfiling {
     repeated EBPFProfilingStackMetadata stacks = 1;
     // stack counts
     int32 dumpCount = 2;
+}
+
+message EBPFOffCPUProfiling {
+    // stack data in one task(thread)
+    repeated EBPFProfilingStackMetadata stacks = 1;
+    // total count of the process is switched to off cpu by the scheduler.
+    int32 switchCount = 2;
+    // where time(microseconds) is spent waiting while blocked on I/O, locks, timers, paging/swapping, etc.
+    int64 duration = 3;
```

Comment:
Later, we may need to evaluate either ms or ns is better, as you are watching event, not periodic sampling.

Reply:
Updated to nanoseconds.

Reply:
Be careful on the backend, I think we provide ms for UI consistently for both ON and OFF CPU.